src/solve2.F

```diff
@@ -2272,7 +2272,7 @@ subroutine solve2(nstep,rbufsz,num_soil_layers,                  &
 #ifdef MPI
       ! finish comms for cflquick:
       call mpi_wait(reqc,mpi_status_ignore,ierr)
-      print *,'solve2: cflmax is: ',cflmax
+      !print *,'solve2: cflmax is: ',cflmax
```

Comment:
could we just delete this line instead of commenting it out?

Reply:
Sounds good to me. Someone else had put it in there to debug, and it was printing out way too much stuff